package.json

```diff
@@ -10,15 +10,15 @@
   },
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/Elerqsousy/webpack.git"
+    "url": "git+https://github.com/Elerqsousy/leaderboard-api"
   },
   "keywords": [],
   "author": "",
   "license": "ISC",
   "bugs": {
-    "url": "https://github.com/Elerqsousy/webpack/issues"
+    "url": "https://github.com/Elerqsousy/leaderboard-api/issues"
   },
-  "homepage": "https://github.com/Elerqsousy/webpack#readme",
+  "homepage": "https://github.com/Elerqsousy/leaderboard-api#readme",
   "dependencies": {
     "webpack": "^5.69.0",
     "webpack-cli": "^4.9.2"
```

Comment:
- [ ] good job adding the webpack and webpack-cli however these packages should be added as devDependencies